api/authenticator.py

```diff
@@ -2594,7 +2595,7 @@ def oauth_authentication_callback_url(cls, library_short_name):
         provider to demonstrate that it knows which URL a patron was
         redirected to.
         """
-        return url_for('oauth_callback', library_short_name=library_short_name, _external=True)
+        return url_for('oauth_callback', library_short_name=library_short_name, _external=True, _scheme='https')
```

Comment:
I'm a little concerned that this could have side effects if partners have OAuth configured and _aren't_ using `https` but that wouldn't be great so I think it's okay?

Reply:
I think clever requires https now... and I think we should as well.  And possibly do automatically on prod? Was a requirement to get this working locally.